ext/opentelemetry-ext-http-requests/tests/test_requests_integration.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+from unittest import mock
+
+import requests
+import opentelemetry.ext.http_requests
+from opentelemetry import trace
+
+class TestRequestsIntegration(unittest.TestCase):
+
+    # TODO: Copy & paste from test_wsgi_middleware
+    def setUp(self):
+        self.span_attrs = {}
+        self.tracer = trace.tracer()
+        self.span_context_manager = mock.MagicMock()
+        self.span = mock.create_autospec(
+            trace.Span, spec_set=True
+        )
+        self.span_context_manager.__enter__.return_value = self.span
+
+        def setspanattr(key, value):
+            self.assertIsInstance(key, str)
+            self.span_attrs[key] = value
+
+        self.span.set_attribute = setspanattr
+        self.patcher = mock.patch.object(
+            self.tracer,
+            "start_span",
+            autospec=True,
+            spec_set=True,
+            return_value=self.span_context_manager
+        )
+        self.start_span = self.patcher.start()
+
+        mocked_response = mock.MagicMock()
```

Comment:
It may be better to not mock this.
```suggestion
        mocked_response = requests.models.Response()
```

Reply:
I did not see any documentation for the constructor of Response, thus I mocked it. But if this is possible then of course it is better to use a "real" Response.